### PR TITLE
ath79-generic: (re)add unifi-ac-{lite,lr}

### PR DESCRIPTION
**Pull request by AiyionPrime**
This is a followup of #2428.

> I think the test of the AC Mesh is sufficient

_@neoraider in hackint#gluon_

### CODE REVIEW

File: package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua [hunk 1]
```diff
@@ -35,6 +35,7 @@ function M.is_outdoor_device()
 		'tplink,cpe510-v3',
 		'tplink,eap225-outdoor-v1',
 		'tplink,wbs210-v2',
+		'ubnt,unifiac-lite',
```
Comment (NeoRaider) [inline package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua:38]: The AC Lite is an indoor device.

File: docs/user/supported_devices.rst [hunk 2]
```diff
@@ -91,6 +91,7 @@ ath79-generic
 
   - UniFi AC Mesh
   - UniFi AP
+  - UniFi AP AC Lite
```
Comment (NeoRaider) [inline docs/user/supported_devices.rst:94]: For consistency with the other UniFi AC variants, this should be "UniFi AC Lite" without the "AP".

File: package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua (hunk 1 above)
Comment (AiyionPrime) [reply package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua]: resolved in d3a2c657f912b1d7354c962fc91bcc5ba437f739

File: docs/user/supported_devices.rst (hunk 2 above)
Comment (AiyionPrime) [reply docs/user/supported_devices.rst]: resolved in d3a2c657f912b1d7354c962fc91bcc5ba437f739

**Comment by rotanid** [comment]
@NeoRaider why did you suggest to change the name of the device in the supported hardware list?
it was called the other way in our ar71xx docs and also in openwrt docs it is called like we did before...
https://github.com/freifunk-gluon/gluon/blob/v2021.1.x/docs/user/supported_devices.rst

**Comment by NeoRaider** [comment]
@rotanid In OpenWrt it's consistently "UniFi AC" now, in both Device Trees and image definitions.

**Comment by rotanid** [comment]
> @rotanid In OpenWrt it's consistently "UniFi AC" now, in both Device Trees and image definitions.

hm, ok, have only looked in the openwrt wiki and TOH.

we need to adjust this PR then, too : https://github.com/freifunk-gluon/gluon/pull/2439

**Pull request merged**